### ISSUE TITLE
Thanks for xmldoc!

### ISSUE THREAD

**Issue by georgyfarniev** [nfarina/xmldoc#61]
Hi, @nfarina , thank you very much for creating this lib, I'm tired to sift through kid's made libraries that just convert xml to json which is inconvenient to traverse, so I'm happy to migrate on xmldoc now.

**Comment by nfarina** [comment]
Very glad you like it!

**Comment by Felipedelima123** [comment]
Yeah, I was struggling with another libs, In some I can convert to JSON and work, but if I needed to convert again to XML, it was breaking the structure. 
This lib really helped me, thanks for creating this!

**Comment by nfarina** [comment]
❤️

**Comment by sophistifunk** [comment]
Yes, thanks from me too, I've used this quite a bit recently! 

I'm going to need to add some support for non-self-closing-but-standalone tags (ie `<img>`, `<br>`, `<col>` etc) in order to parse html. Are you interested in me making a PR for it here when I'm done, or should I just fork?

**Comment by nfarina** [comment]
I'd be happy to review a PR for that! I think you're limited to what [sax](https://github.com/isaacs/sax-js) can provide though, I'd start there.

**Comment by sophistifunk** [comment]
I looked into it and it required too many changes to the underlying sax lib, and the replacement for that lib is even more focused on document correctness. In the end I had to go with [htmlparser2](https://github.com/fb55/htmlparser2) for this use case. I recommend it for anybody in a similar situation, as the object model is quite similar. But I'll still be sticking with `xmldoc` when I'm expecting well-formed documents.
